src/types.test.ts

```diff
@@ -1,4 +1,4 @@
-import { DSNP, DSNPMessage } from "./types/DSNP";
+import * as DSNP from "./types/DSNP.d";
```

Comment:
Is the `.d` really needed for it to pick it up? It shouldn't be if I'm reading the module resolution docs correctly, but perhaps it depends on the type of module resolution we are using?
https://www.typescriptlang.org/docs/handbook/module-resolution.html#module-resolution-strategies

Reply:
I forgot about that; tsc was not finding the file.  This was a suggestion from tsc, but once I got the other things in place I forgot to take it out and try it.

Reply:
For future reference: We figured out that it was the `export const`s in the definition file that were the issue. Since .d.ts files are not compiled, those variables would disappear. Moved those out to a separate file and then we could remove the .d from the import and pull the types in.